### ISSUE TITLE
monetary type

### ISSUE THREAD

**Issue by sbidoul** [OCA/mis-builder#20]
A monetary kpi type which behaves exactly as the numeric type, except it renders with the proper monetary sign.

**Comment by sbidoul** [comment]
@Martronic-SA you had a branch somewhere with that, did you?

**Comment by Martronic-SA** [comment]
yes, here it is [#5](https://github.com/Martronic-SA/account-financial-reporting/pull/5)
basically this commit [#9b220a8c155d1f68e9486cb05421bb83dbcce622](https://github.com/Martronic-SA/account-financial-reporting/commit/9b220a8c155d1f68e9486cb05421bb83dbcce622)
